packages/react-core/src/components/Toolbar/ToolbarContent.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Toolbar/toolbar';
+import flexStyles from '@patternfly/react-styles/css/layouts/Flex/flex';
```

Comment:
re: https://github.com/patternfly/patternfly-react/issues/8364#issuecomment-1400998779

Yep, you should be able to get these styles from the toolbar. Looks like v5 is on core `5.0.0-alpha.3` https://github.com/patternfly/patternfly-react/blob/e10574726614bbe29582d3212d04f2d3d28be6ca/packages/react-core/package.json#L57

https://github.com/patternfly/patternfly-react/blob/e10574726614bbe29582d3212d04f2d3d28be6ca/packages/react-styles/package.json#L22

@mattnolting's toolbar updates are in `5.0.0-alpha.3` 
https://github.com/patternfly/patternfly/releases/tag/prerelease-v5.0.0-alpha.6. Are you able to pull that in?

Reply:
@mcoker  - The toolbar.d.ts file does not have any mention of `center` at all. I was thinking there would be `modifiers.alignItemsCenter` for example.

I do see that the `toolbar.css` which that file imports has `.pf-c-toolbar {  --pf-c-toolbar--m-full-height__item--AlignItems: center; }`, but I'm assuming this is just a style that's automatically getting applied with full height rather than an option i can trigger here.

Reply:
@gitdallas doh! Yep I meant to say alpha.6, you can see the compiled classes here https://unpkg.com/browse/@patternfly/patternfly@5.0.0-alpha.6/components/Toolbar/toolbar.css